dispatch/apps/api/serializers.py

```diff
@@ -14,13 +14,46 @@ class UserSerializer(DispatchModelSerializer):
     """
     Serializes the User model.
     """
+
+    email = serializers.EmailField(required=True)
+    person_id =  serializers.IntegerField(validators=[validate_person_id])
+    password_a = serializers.CharField(
+        required=False, 
+        write_only=True, 
+        validators=[PasswordValidator(confirm_field='password_b')]
+    )
+    password_b = serializers.CharField(required=False, write_only=True)
+
     class Meta:
         model = User
         fields = (
             'id',
-            'email'
+            'email',
+            'person_id',
+            'password_a',
+            'password_b',
         )
 
+    def create(self, validated_data):
+
+        instance = User()
+        if User.objects.filter(email=validated_data['email']).exists():
```

Comment:
I'm not quite sure how to go about this @psiemens. When we wanted to deal with the Person database error for cascading deletions we changed the PersonViewSet perform_destroy method, in this case should I override the UserViewSet perform_create method in [apps/api/views.py](views file)?